ci-operator/config/openshift/boilerplate/openshift-boilerplate-master.yaml

```diff
@@ -1,24 +1,10 @@
-base_images:
-  base:
-    name: "4.4"
-    namespace: ocp
-    tag: base
-  openshift_release_golang-1.13:
-    name: release
-    namespace: openshift
-    tag: golang-1.13
 build_root:
-  project_image:
-    context_dir: config/
-    dockerfile_path: Dockerfile
+  from_repository: true
 images:
 - context_dir: config/
   dockerfile_path: Dockerfile
-  from: openshift_release_golang-1.13
+  from: src
```

Comment:
I assume this means use the `.ci-operator.yaml` to get the base image.

Reply:
Yeah, `src` is a special name given to the image produced by `build_root`. Or is it `root`? I think (per the [docs](https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-artifacts)) that either will work here, but let me check with dptp...

Reply:
`src` is `root` with the repository cloned into it.

Reply:
Cool, that's what we want here ✓

Thanks @petr-muller.